### PR TITLE
Add an ability to specify manifest changes

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Fixes: #1173 

## This introduces a breaking change

- [x] No

### CODE REVIEW

**Review by harshad16 — COMMENTED** [review]
lgtm

File: thoth/adviser/state.py [hunk 1]
```diff
@@ -44,6 +44,19 @@ class State:
     resolved_dependencies = attr.ib(default=attr.Factory(dict))  # type: Dict[str, Tuple[str, str, str]]
     _parent = attr.ib(default=None)  # type: weakref.ReferenceType['State']
     advised_runtime_environment = attr.ib(type=Optional[RuntimeEnvironment], kw_only=True, default=None)
+    # an example for a manifest change:
```
Comment (goern) [inline thoth/adviser/state.py:47]: is this inspired or just like how kustomize uses jsonpatch? see https://github.com/kubernetes-sigs/kustomize/blob/master/examples/jsonpatch.md

Comment (fridex) [reply thoth/adviser/state.py:47]: it's inspered, however we could use https://pypi.org/project/jsonpatch/ might be a better option

**Comment by goern** [comment]
@harshad16 could you have a look why the pytest results are cut off? thx!

**Comment by harshad16** [comment]
> @harshad16 could you have a look why the pytest results are cut off? thx!

may be it was a flake earlier , however with previous `/retest` it gave positive result.

**Pull request merged**